src/widgets/numeric-menu/numeric-menu.tsx

```diff
@@ -135,7 +133,7 @@ export type NumericMenuTemplates = {
     /**
      * The CSS classes provided to the widget.
      */
-    cssClasses: NumericMenuRendererCSSClasses;
+    cssClasses: NumericMenuComponentCSSClasses;
```

Comment:
alternatively, like here there's indeed a use for the classes specifically passed by this widget, so it's also all right for the version you have today